notebooks/Emukit-basic-use-of-the-library.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basic use of the Emukit\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# General imports and parameters of figures should be loaded at the beginning of the overview\n",
+    "\n",
+    "### General imports\n",
+    "%matplotlib inline\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import colors as mcolors\n",
+    "\n",
+    "### --- Figure config\n",
+    "colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)\n",
+    "LEGEND_SIZE = 15\n",
+    "TITLE_SIZE = 25\n",
+    "AXIS_SIZE = 15"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to illustrate how the a model can wrapped and used in different tasks in Emukit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Navigation\n",
+    "\n",
+    "1. [Load your objective, collect some data, build a model](#1.-Load your objective, collect some data, build a model)\n",
+    "\n",
+    "2. [Load the elements to solve your problem and run the decision loop if needed](#2.-Load the elements to solve your problem and run the decision loop if needed)\n",
+    "\n",
+    "3. [Conclusions](#3.-Conclusions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.  Load your objective, collect some data, build a model\n",
+    "\n",
+    "These steps are common to all methods in Emukit.Here we illustrate how to do it with the branin function."
```

Comment:
"...ethods in Emukit.Here we illustrate how..." -> whitespace missing after dot

Reply:
corrected